framework/src/media/utils/MediaUtils.cpp

```diff
@@ -322,14 +322,13 @@ bool header_parsing(FILE *fp, audio_type_t audioType, unsigned int *channel, uns
 				ret = fseek(fp, -1, SEEK_CUR);
 				if (ret != OK) {
 					meddbg("file seek failed errno : %d\n", errno);
-					free(header);
 					return false;
 				}
 			}
 		}
 		if (isHeader) {
 			header = (unsigned char *)malloc(sizeof(unsigned char) * (MP3_HEADER_LENGTH + 1));
-			
+
```

Comment:
why do you add a empty line here?

Reply:
^^, I'm not adding a empty line.., but `remove extra white spaces when saving`.

Reply:
OK，my mistake.  Its look good to me.   :)